packages/appstore/src/components/fiat-transaction-list/fiat-transaction-list.tsx

```diff
@@ -25,6 +25,7 @@ const FiatTransactionList = () => {
         account_type: 'binary',
         balance: 0,
         currency: 'BTC',
+        gradient_for: 'btc',
```

Comment:
@hamid-deriv Why do you need to define new key(grandient_to) since this is same as currency  with lowercase characters?